### DISCUSSION TITLE
How to implement options for comments collection

### DISCUSSION THREAD

**Discussion by Rhodungeon** [kevinzg/facebook-scraper#209] [Q&A] [answered]
Hi, great tool, very useful for my project!
I was wondering how to collect also posts'comments... it seems that this possibility is already available as it appears as marked in the "to do" list section

- [ ] Comments (with options={'comments': True})

Thank you for your help

**Comment by neon-ninja** [comment, marked as answer]
Hi, you can find an example here - https://github.com/kevinzg/facebook-scraper/pull/188

**Reply by Rhodungeon** [reply]
Thank you!
